_includes/scripts.html

```diff
@@ -11,19 +11,7 @@
 
 {% if site.ga.ua %}
 <!-- Google Analytics -->
-<script async src="https://www.googletagmanager.com/gtag/js?id={{ site.ga.ua }}"></script>
-<script>
-  window.dataLayer = window.dataLayer || [];
-  function gtag() { dataLayer.push(arguments); }
-  gtag('js', new Date());
-
-  /**
-    * `forceSSL` was used for analytics.js (the older Google Analytics script).
-    * It isn't documented for gtag.js, but the term occurs in the gtag.js code;
-    * figure it doesn't hurt to leave it in. -@afeld, 5/29/19
-    */
-  gtag('config', '{{ site.ga.ua }}', { 'anonymize_ip': true, 'forceSSL': true });
```

Comment:
Note to self: Do we want to keep `anonymize_ip`? I'm fine with removing `forceSSL` here since it was of dubious utility back in 2019 (according to the comment) and it's probably even less so now.